src/frontend-pwa/src/App.tsx

```diff
@@ -16,6 +16,7 @@ import constants from './constants/Constants';
 
 function App() {
   const [isLoading, setIsLoading] = useState(true);
+  // you have set the following value to true for testing
```

Comment:
How about having an environment variable called TESTING and then you can right it as `useState(Boolean(process.env.TESTING) ? true : false)`

Reply:
@BradyMitch Once state management is set up (this sprint) The state will be properly used to display the load screen